toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexStartupActivity.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.crates.local
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.startup.StartupActivity
+import com.intellij.openapiext.isUnitTestMode
+import org.rust.ide.experiments.RsExperiments
+import org.rust.openapiext.isFeatureEnabled
+
+class CratesLocalIndexStartupActivity : StartupActivity {
+    override fun runActivity(project: Project) {
+        if (!isUnitTestMode && isFeatureEnabled(RsExperiments.CRATES_LOCAL_INDEX)) {
+            CratesLocalIndexService.getInstance()
```

Comment:
AFAIU this call only initialize but does not actually **build** the index. To ultimately build it, `CratesLocalIndexService#updateIfNeeded` should be invoked.

Also, if something heavyweight like `CratesLocalIndexService#updateIfNeeded` is called, we may consider `StartupActivity.Background` instead of `StartupActivity`